pkg/planner/core/indexmerge_path.go

```diff
@@ -180,6 +180,10 @@ func (ds *DataSource) generateNormalIndexPartialPaths4DNF(dnfItems []expression.
 }
 
 // getIndexMergeOrPath generates all possible IndexMergeOrPaths.
+// For index merge union case，the order property from its partial
+// path can be kept and multi-way merged and output. So we don't
+// generate a concrete index merge path out, but an un-determined
+// alternatives set index merge path instead.
```

Comment:
Could you add an example here? For example, for `select ... order by ...`, the `partialAlternativePaths` can be `...`.